modules/apps/batch-engine/headless/headless-batch-engine-impl/rest-openapi.yaml

```diff
@@ -0,0 +1,113 @@
+components:
+    schemas:
+        Task:
+            properties:
+                className:
+                    example: com.liferay.headless.delivery.dto.v1_0.BlogPosting
+                    type: string
+                endTime:
+                    example: 1569261245
```

Comment:
why not a date? it's an amount in milliseconds?

Reply:
It is the amount in milliseconds

Reply:
mmm we usually use dates in ISO 8601 format